packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2307,6 +2307,63 @@ abstract class BuildContext {
 /// To assign a build owner to a tree, use the
 /// [RootRenderObjectElement.assignOwner] method on the root element of the
 /// widget tree.
+///
+/// {@tool dartpad --template=freeform}
+/// This example shows how to build an off-screen widget tree used to measure
```

Comment:
For some use cases, the Offstage widget may offer a simpler alternative to this. Should we mention that here?